### ISSUE TITLE
Missing labels / accessible names for input and select elements

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-GuestUser#33]
A few missing labels / names on Browse Users:
- The search text `input` is missing a label.
- `.quick-filter` `select` element is missing an accessible name.

**Comment by kimisgold** [comment]
Aria labels should be in for these now.

**Comment by aviannamiller** [comment]
Thanks, those are fixed.
